v2/kafka-common/src/main/java/com/google/cloud/teleport/v2/kafka/options/KafkaReadOptions.java

```diff
@@ -49,23 +49,49 @@ public interface KafkaReadOptions extends PipelineOptions {
 
   void setKafkaReadTopics(String inputTopics);
 
-  @TemplateParameter.Enum(
+  @TemplateParameter.Boolean(
       order = 3,
       groupName = "Source",
+      name = "enableCommitOffSets",
+      optional = true,
+      description = "Enable committing offsets to Kafka",
+      helpText =
+          "Enable KafkaIO to commit offsets after consuming the record. Enabling this requires you to provide"
```

Comment:
I suggest changing the help text to:

"Commit offsets of processed messages to Kafka. If enabled, this will minimize the gaps or duplicate processing of messages when restarting the pipeline. Requires specifying the Consumer Group ID."

because:

1) Users of the template may not know what "KafkaIO" means, better to not mention it.
2) "or the template launcher will fail" - an unnecessary detail.
3) Added some more details to explain why this may be useful ("minimize the gaps ..." is copied from KafkaIO javadoc).

Reply:
Yep. this sounds good to me. Thanks